src/_common/Tab.js

```diff
@@ -21,13 +21,13 @@ export default class Tab extends Component {
     };
 
     render() {
-        const { imgSrc, selected, showIcon, showText, text, onMouseDown, onClose, closable } = this.props;
+        const { imgSrc, selected, showIcon, showText, text, onClick, onClose, closable } = this.props;
 
         return (
             <div
                 role="tab"
                 aria-selected={selected}
-                onMouseDown={onMouseDown}
+                onClick={onClick}
```

Comment:
@borisyankov 

Do you remember why onMouseDown was use?
It causes a subtle bug on dropdown, explained as follow.

    When you click on children of dropdown, some event is fired on 
    the dom element you clicked, then if the dom element 
    shrink in sizes, suddenly the event will be fired on the 
    full-screen-overlay, thus trigger dropdown onclose method.

After an hour of searching the web, accidentally I found this code (god bless) 
do you think this changes will break anything?